blocks/career-carousel/career-carousel.js

```diff
@@ -0,0 +1,158 @@
+import { createOptimizedPicture } from '../../scripts/lib-franklin.js';
+import { fetchIndex } from '../../scripts/scripts.js';
+
+export function filterIncompleteEntries(json) {
+  return json.data.filter((e) => e.image !== '' && e['career-quote'] !== '0' && e['career-jobtitle'] !== '0');
+}
+
+export function scrollToCard(idx, card, precedingCard, slides, span, doc) {
+  const rect = card.getBoundingClientRect();
+
+  // set the style on the active button
+  const buttons = doc.querySelectorAll('.career-slides-nav span.active-nav');
+  buttons.forEach((b) => b.classList.remove('active-nav'));
+  span.classList.add('active-nav');
+
+  // Compute the gap to add to the location
+  let gap = 0;
+  if (precedingCard) {
+    const prevRect = precedingCard.getBoundingClientRect();
+    gap = rect.x - (prevRect.x + prevRect.width);
+  }
+
+  slides.scrollTo(idx * (rect.width + gap), slides.scrollHeight);
+}
+
+export function scrollToAdjacent(spans, slideDivs, slides, next, doc) {
+  let curActive;
+  for (let i = 0; i < spans.length; i += 1) {
+    if (spans[i].classList.contains('active-nav')) {
+      curActive = i;
+      break;
+    }
+  }
+
+  if (curActive === undefined) {
+    return;
+  }
+
+  let newActive = curActive + (next ? 1 : -1);
+  if (newActive < 0) {
```

Comment:
can we directly use modulo operator here
something like
newActive = (newActive + spans.length) % spans.length

Reply:
A little cryptic but yes and it would allow me to turn `newActive` into a const so I've made this change.